modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc

```diff
@@ -110,8 +121,36 @@ image::provision-service-mesh-adapter-output.png[Example Output]
 .. Using the deploy name from the previous command, restart all your services that need to be managed by Anypoint Service Mesh to inject the sidecar in the pod:
 +
 `kubectl -n <adapter-namespace> patch deploy <deployname> --type=json -p='[{"op": "replace", "path": "/spec/template/metadata/labels/service-mesh.mulesoft.com","value":"enable"}]'`
-. To check whether the applications are redeployed, run the following command:
+. To verify that the the applications are redeployed, run the following command:
 +
 `kubectl get pods -n <namespace>`.
 
+[[provision_using_CRD]]
+== Provisioning the Adapter Using the CRD File 
+
+To provision the Anypoint Service Mesh adapter using the CRD file:
+
+. Create your adapter definition file and save it with a '.yaml' file extension:
++
+[source,text,linenums]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: <adapter name> # example: bookinfo-service-mesh-adapter
+  namespace: <namespace name> # example: default
+spec:
+  clusterServiceClassExternalName: anypoint-service-mesh-adapter
+  clusterServicePlanExternalName: <plan size > # example: small.
+  parameters:
+    clientId: <client id> # example: 4bc55ee4cfc84a2ddddbb2c1109d1123c5c4
+    clientSecret: <client secret> # example: 8E5ae17BbB664Eddd9ab32A5fA869874Be7
+    platformUri: <platform uri> # example: https://eu1.anypoint.mulesoft.com
```

Comment:
Please check that I had added two fields on the doc:
```
    replication:
      replicas: <amount of replicas> # example: 2
```

Reply:
Added a comment in the doc. Need clarity.